retail-managment-system/rms-ui/src/app/domain/installment/models/installment.ts

```diff
@@ -0,0 +1,13 @@
+import { Order } from "../../order/models/order";
+
+
+export interface Installment {
+     id: number;
+    version: number;
+    installmentAmount :number;
+    paymentAmount : number;
+    dueDate : string;
+    paymentDate : string;
+    status : number ; 
+    orderDTO : Order   
```

Comment:
orderDTO is not the same property in swagger.

Reply:
Done